viewpump/src/main/java/io/github/inflationx/viewpump/ReflectiveFallbackViewCreator.java

```diff
@@ -0,0 +1,52 @@
+package io.github.inflationx.viewpump;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.AttributeSet;
+import android.view.View;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+
+class ReflectiveFallbackViewCreator implements FallbackViewCreator {
+
+    private static final Class<?>[] constructorSignature2 = new Class[] {
```

Comment:
I hadn't added a `ViewPump.create()` overload that would use them, though i guess I could. also, guess i could look for the 3-param or 4-param, in case the custom view doesn't have a 1 or 2 param